### PR TITLE
Final round of changes before the release of 0.0.1

### PR DESCRIPTION
**Pull request by thibautjombart**
This should be closing https://github.com/whocov/trend_analysis_public/issues/25

Only part left out is adding vaccination coverage, now put in a separate issue: https://github.com/whocov/trend_analysis_public/issues/26

### CODE REVIEW

File: report_sources/elr_review.Rmd [hunk 1]
```diff
@@ -147,17 +147,18 @@ rate (*r*), current incidence (as % of previous peak) and signs of acceleration
 
 * if incid < 40%:
     + if r > high_r -> **M**
-	+ else -> NL
+    + else if r > 0.02 -> **L**
+	+ else -> Minimal
 * if 40% <= incid <= 80%:
```
Comment (nsbatra) [inline report_sources/elr_review.Rmd:152]: What style of coding is this? Just curious

**Review by nsbatra — APPROVED** [review]
Looks good to me. Thanks for making all these annoying verbiage changes

**Comment by henryls1** [comment]
looks good to me!

**Review by henryls1 — APPROVED** [review]

File: report_sources/elr_review.Rmd (hunk 1 above)
Comment (thibautjombart) [reply report_sources/elr_review.Rmd:152]: That was my attempt at pseudo-code, but I never learnt properly how to do this...

**Pull request merged**